.github/workflows/docker_release.yml

```diff
@@ -1,19 +1,22 @@
 name: Docker image release
 
 on:
-  workflow_dispatch:
   push:
     branches:
       - main
-    tags:
-      - "v[0-9].[0-9]+.[0-9]+*"
+  workflow_run:
+    workflows: [Project release on PyPi]
```

Comment:
Assuming the name must match, can we add a comment in `.github/workflows/pypi_release.yml` warning that changing that workflow name will break this one?